assets/scss/layouts/header/_header.scss

```diff
@@ -132,6 +131,16 @@
     }
 }
 
+.header-logo-position-left {
+    float: left;
+    margin-left: remCalc(40px);
+}
+
+.header-logo-position-right {
+    float: right;
+    margin-right: remCalc(40px);
+}
+
```

Comment:
That overlap of the logo over 'CART' can be removed with the `margin-right` value you add for   `header-logo-position-{right/left}`